app/examples/vimium

```diff
@@ -3,6 +3,7 @@ set adblocker=update
 set downloadmethod=confirm
 set mintabwidth=250
 set norestoretabs
+set follownewtabswitch
```

Comment:
This will enable the toggle, which is already the default, I think you are looking to disable it instead? You can do this using `set nofollownewtabswitch`.